java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -56,6 +57,8 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
 
         baseTransaction.set(key2, value2, SetOptions.builder().returnOldValue(true).build());
         baseTransaction.strlen(key2);
+        baseTransaction.append(key10, value1);
+        baseTransaction.get(key10);
```

Comment:
According to the Python test, they use key2 here. And double the length of the string there. 

Note: then they have a mset call - which you could include. 

```
        baseTransaction.append(key2, value1);
        baseTransaction.mset(Map.of(key1, value2, key2, value1));
```

and the result... 
```
            Long.valueOf(value1.length() * 2), // append(key2, value1)
            OK, // mset(Map.of(key1, value2, key2, value1)
```

https://github.com/Bit-Quill/glide-for-redis/blob/main/python/python/tests/test_transaction.py#L72-L73

Reply:
@acarbonetto when I add in your new implementation, it causes a test in the TransactionTests.java to break because it's now expecting a different value than what it initially had since you wrote this test before I wrote the append method. I think it's best to add another key to avoid any of these test conflicts. Thoughts?